mne/io/base.py

```diff
@@ -847,6 +853,15 @@ def get_data(self, picks=None, start=0, stop=None,
         -----
         .. versionadded:: 0.14.0
         """
+        # validate types
+        _validate_type(start, types='int', item_name='start')
+        if stop is not None:
+            _validate_type(stop, types='int', item_name='stop')
```

Comment:
@larsoner `"int"` works because of these lines: https://github.com/mne-tools/mne-python/blob/3843c36cec7310eb7b7c793e825be608f81af852/mne/utils/check.py#L391-L393

Reply:
I amended the docstr of _validate_type to mention `'int-like'` as an option for the `types` parameter.

```
    types : type | str | tuple of types | tuple of str
         The types to be checked against.
         If str, must be one of {'int', 'int-like', 'str', 'numeric', 'info',
         'path-like', 'callable'}.
         If a tuple of str is passed, use 'int-like' and not 'int' for
         integers.
```